rules.yaml

```diff
@@ -222,6 +238,16 @@ features:
           target: examples/appthreadsafe/conanfile.txt
         - source: examples/appthreadsafe/main.cpp.tpl
           target: examples/appthreadsafe/main.cpp
+  - name: examples_olink
+    requires:
+      - api
+      - core
+      - stubs
+      - apigear
```

Comment:
you should not need to require `apigear` explicitly here, that should be added by `olink` implicitly.
Same probably goes for all other feature dependencies to `olink`, as it already requires `api`, `core`,  and `apigear`
Only `stubs` is needed here besides `olink`